tools/go-cli/go-whisk-cli/commands/commands.go

```diff
@@ -76,70 +76,57 @@ func init() {
     }
 }
 
+func getKeyValueArgs(args []string, argIndex int, parsedArgs []string) ([]string, []string, error) {
+    var whiskErr error
+
+    if len(args) - 1 >= argIndex + 2 {
+        parsedArgs = append(parsedArgs, args[argIndex + 1])
+        parsedArgs = append(parsedArgs, args[argIndex + 2])
+        args = append(args[:argIndex], args[argIndex + 3:]...)
+    } else {
+        whisk.Debug(whisk.DbgError, "%s arguments must be a key/value pair; args: %s", args[argIndex], args)
+        errMsg := fmt.Sprintf("%s arguments must be a key/value pair", args[argIndex])
```

Comment:
I'm thinking that `args[argIndex]` is a single argument.. If so, in lines 87,88 "%s arguments" -> "argument '%s'".